java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/GenericMqttConnectionConfig.java

```diff
@@ -0,0 +1,23 @@
+package com.cumulocity.generic.mqtt.client;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class GenericMqttConnectionConfig {
+
```

Comment:
suggestion: No need of the '_IN_' part IMO, but happy to be disagreed with. 
That is, change DEFAULT_CONNECTION_TIMEOUT_IN_MILLIS to DEFAULT_CONNECTION_TIMEOUT_MILLIS